packages/core-webhooks/src/conditions.ts

```diff
@@ -1,12 +1,16 @@
-export const between = (actual, expected): boolean => actual > expected.min && actual < expected.max;
+import { Utils } from "@arkecosystem/crypto";
+
+const toBoolean = (value): boolean => value.toString().toLowerCase().trim() === "true" ? true : false;
+
+export const between = (actual, expected): boolean => gt(actual, expected.min) && lt(actual, expected.max);
 export const contains = (actual, expected): boolean => actual.includes(expected);
 export const eq = (actual, expected): boolean => actual === expected;
```

Comment:
I think we should check here if the given values are potentially in the bignumber range and then compare them as such.